### DISCUSSION TITLE
full network version

### DISCUSSION THREAD

**Discussion by ducky1241** [AdguardTeam/AdGuardHome#7124] [Q&A]
I am currently running Adguard Home on my Ubuntu box. I am currently using it for protection on my full network. It works fine for me. I normally have one Windows 11 box that pretty much runs 24/7.  I was wondering if there was an Adguard Home version I can run on my Windows box that will protect my entire network. If so where can I download it

**Comment by ppfeufer** [comment]
Scroll down to the windows downloads here » https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.52
